Editor/Core/iTextureAnalyzer.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using UnityEditor;
+namespace Unity.ProjectAuditor.Editor.Core
+{
+    public interface ITextureAnalyzer
+    {
+        void Initialize(ProjectAuditorModule module);
+        IEnumerable<ProjectIssue> Analyze(BuildTarget platform, TextureImporter currentimporter, TextureImporterPlatformSettings tips);  // Unsure if TextureImporterSettings or EditorUserBuildSettings (EditorUserBuildSettings.androidBuildSubtarget) needed
```

Comment:
Not sure if : 
Analyze(BuildTarget platform, TextureImporter currentimporter, TextureImporterPlatformSettings tips);
has the all parameters we want to include, or has some we don't want to include....

Reply:
If anything is missing we will find out once we implement a `TextureAnalyzer`